docs/docs/rest-api/public/api/v2/types/healthCheck.raml

```diff
@@ -148,3 +148,15 @@ types:
         description: Amount of time to wait until starting the health checks.
         minimum: 0
         default: 15
+  Health:
+    type: object
+    properties:
+      alive: boolean
+      consecutiveFailures:
+        type: number
+        format: int32
+      firstSuccess?: string
```

Comment:
isn't this a `datetime`? Or is the format different? Compare e.g. to `app.version`.